### DISCUSSION TITLE
Embedded Swift

### DISCUSSION THREAD

**Discussion by YouCanNotBeSerious** [HomeSpan/HomeSpan#884] [General]
https://developer.apple.com/videos/play/wwdc2024/10197/

Embedded Swift on an ESP32C6 as a HomeKit accessory.... what do we think, what's it mean for HomeSpan?

**Comment by frankonski** [comment]
2c. Swift is its own language. HomeSpan is in C++. Developers have the ability to choose their language and library. HomeSpan is just wonderful. I love C, I know jack about swift and not interested either. I’ll stick with HomeSpan.
